tests/integration/conftest.py

```diff
@@ -69,3 +69,23 @@ def local_pypiserver() -> Iterator[None]:
             raise RuntimeError("pypiserver didn't start fast enough")
 
         yield
+
+
+def does_not_test_any_affected_pms(test: Any, affected_pms_to_test: set[str]) -> bool:
+    pm_under_test = Path(test.fspath).stem.lstrip("test_")
+    return pm_under_test not in affected_pms_to_test
+
+
+def pytest_collection_modifyitems(session: Any, config: Any, items: Any) -> None:
+    skip_mark = pytest.mark.skip(reason="No changes to tested code")
+    to_test = utils.determine_integration_tests_to_run_for("HEAD")
```

Comment:
Suggestion:
https://docs.pytest.org/en/7.1.x/_modules/_pytest/hookspec.html#pytest_collection_modifyitems
I found type annotations and docstring for this function in pytest documentation. We can borrow that or get some inspiration.

Reply:
> We can borrow that or get some inspiration.

We should use it **verbatim** as is in the docs.

Reply:
Could you add a short docstring as well please ?

Reply:
Sure, done.